### PR TITLE
heading level and doc for masonry readme

### PR DESCRIPTION
**Pull request by KGrewal1**
Update the Masonry readme with documentation on how it is generated and change the heading level to be consistent with Color

### CODE REVIEW

File: masonry/README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@
 
 [tracing_tracy]: https://crates.io/crates/tracing-tracy
```
Comment (DJMcNab) [inline masonry/README.md:17]: This line is the equivalent of the `libm` line in: https://github.com/linebender/color/blob/9775885b74e37f2de2a1713f80c607ba84e474c1/color/README.md#L21-L23

Therefore, it should be below the new comment and the links block should have the same comment as in color.

**Review by DJMcNab — APPROVED** [review]
Looks good! Again, let me know when you're ready for this to merge.

**Comment by KGrewal1** [comment]
Fine with merging now if this looks good to you!

**Comment by KGrewal1** [comment]
Made change to add the comment, so good to merge on my side!

**Pull request merged**